app/views/posts/index.html.erb

```diff
@@ -2,7 +2,7 @@
   <div id='post-<%= post.id %>' class="post">
     <div class="post__head">
       <span class="post__head__user">
-        <%= post.id  %>
+        <%= post.first_name + " " + post.last_name  %>
```

Comment:
If this is the intended output, maybe use one method on the model that returns the full name?

And TEST it 🎉